src/pages/managerForm.js

```diff
@@ -1,66 +1,387 @@
+                        <input 
+                            type='text' name='community_manager' id='community_manager' value={formData.community_manager}
+                            onChange={handleChange}
+                            required className={styles.inputStyle}
+                        />
+                    </div>
+                    <div className={styles.layoutRow}>
+                        <label 
+                            htmlFor='community_date'
+                        >Date Established
+                        </label>
+                        <input 
+                            type='text' 
+                            name='community_date' 
+                            id='date' 
+                            value={formData.community_date} onChange={handleChange}
+                            className={styles.inputStyle} 
+                        />
+                    </div >
+                    <div className={styles.layoutRow}>
+                        <label  
+                            htmlFor='community_location'
+                        >Location
+                        </label>
+                        <input 
+                            type='text' 
+                            name='community_location' 
+                            id='location' 
+                            value={formData.community_location} onChange={handleChange} 
+                            className={styles.inputStyle} 
+                        />
+                    </div>
+                    <div className={styles.layoutRow}>
+                        <label 
+                            htmlFor='community_url'
+                        >Community URL
+                        </label>
+                        <input 
+                            type='url' 
+                            name='community_url' 
+                            id='community_url' 
+                            value={formData.community_url} 
+                            onChange={handleChange} placeholder='https://example.com' pattern='https://.*' size='50'  required className={styles.inputStyle} 
+                        />
+                    </div>
+                    <div className={styles.layoutRow}>
+                        <label 
+                            htmlFor='community_linkedin'
+                        >Community LinkedIn
+                        </label>
+                        <input 
+                            type='text' 
+                            name='community_linkedin' id='social_url' 
+                            value={formData.community_linkedin} onChange={handleChange} 
+                            className={styles.inputStyle} 
+                        />
+                    </div >
 
-                <div className={styles.communityGrid}>
+                    <div className={styles.layoutRow}>
+                        <label htmlFor='community_twitter'
+                        >Community Twitter
```

Comment:
Actually the Twitter URL is for the Community Manger, not the Community.
Move this field to right after the Community Manager LinkedIn field.

Reply:
Done